tools/ci_build/github/windows/extract_nuget_files_gpu.ps1

```diff
@@ -4,21 +4,21 @@
 # This file is used by Zip-Nuget-Java Packaging Pipeline
 New-Item -Path $Env:BUILD_BINARIESDIRECTORY\RelWithDebInfo\RelWithDebInfo\nuget-artifacts -ItemType directory
```

Comment:
Space only changes?

Reply:
line 32 is the only change. Some others are just white space changes.